homeassistant/components/assist_pipeline/pipeline.py

```diff
@@ -1024,39 +1024,38 @@ async def text_to_speech(self, tts_input: str) -> str:
             )
         )
 
-        try:
-            # Synthesize audio and get URL
-            tts_media_id = tts_generate_media_source_id(
-                self.hass,
-                tts_input,
-                engine=self.tts_engine,
-                language=self.pipeline.tts_language,
-                options=self.tts_options,
-            )
-            tts_media = await media_source.async_resolve_media(
-                self.hass,
-                tts_media_id,
-                None,
-            )
-        except Exception as src_error:
-            _LOGGER.exception("Unexpected error during text-to-speech")
-            raise TextToSpeechError(
-                code="tts-failed",
-                message="Unexpected error during text-to-speech",
-            ) from src_error
-
-        _LOGGER.debug("TTS result %s", tts_media)
+        if tts_input.strip():
```

Comment:
Should you store stripped output ? 
```suggestion
        if tts_input := tts_input.strip():
```